src/providers/historical-telemetry-provider.js

```diff
@@ -40,48 +40,37 @@ export default class YamcsHistoricalTelemetryProvider {
         return this.getHistory(domainObject.identifier.key,
             options.start,
             options.end,
-            options.size);
+            options.size,
+            options.strategy);
     }
 
-    getHistory(id, start, end, size=300) {
+    getHistory(id, start, end, size=300, strategy) {
         // cap size at 1000, temporarily to prevent errors
         if (size > 1000) {
             size = 1000;
         }
 
         let url = this.url + 'api/archive/' + this.instance + '/parameters' + idToQualifiedName(id);
-        url += '?start=' + (new Date(start).toISOString());
-        url += '&stop=' + (new Date(end).toISOString());
-        url += '&limit=' + size;
-        url += "&order=asc";
-
-        return fetch(encodeURI(url))
-            .then(res => {return res.json();})
-            .then(res => {
-                if (!(res.continuationToken)) {
-                    return this.convertPointHistory(id, res);
-                } else {
-                    return this.getSampleHistory(id, start, end, size);
-                }
-            });
-    }
-
-    getSampleHistory(id, start, end, size=300) {
-        // cap size at 1000, temporarily to prevent errors
-        if (size > 1000) {
-            size = 1000;
+        let order = 'asc';
+        let sizeParam = 'limit';
+        let convertHistory = (res) => this.convertPointHistory(id, res);
+        if (strategy && strategy.toLowerCase() === 'latest') {
+            size = 1;
+            order = 'desc'
+        } else if (strategy && strategy.toLowerCase() === 'minmax') {
+            url += '/samples';
+            sizeParam = 'count';
+            convertHistory = (res) => this.convertSampleHistory(id, res);
         }
 
-        let url = this.url + 'api/archive/' + this.instance + '/parameters' + idToQualifiedName(id);
-        url += '/samples';
         url += '?start=' + (new Date(start).toISOString());
         url += '&stop=' + (new Date(end).toISOString());
```

Comment:
Let's use template strings for the rest of this URL as well, just to be consistent.